docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,36 +1,150 @@
 # Parameters
```

Comment:
As a reviewer, let me ask one very basic and high level question:

What is DevOps Dojo?
(The related links are bit too long for me to go through in a short period of time, so a short summary of its idea is very much appreciated, thanks!)

Reply:
@kminoda 

[DevOps Dojos](https://docs.google.com/presentation/d/1hsLgp1myeKhelRjOedA5yi-UK6ZNfwAUxRbvxdSdX_g/edit?usp=sharing) is the way we structure work in the Open AD Kit WG. Each dojo addresses a certain topic, in this case the ROS parameters.

Find our Kanban board [here](https://github.com/orgs/autowarefoundation/projects/6/views/4).